doc/source/conf.py

```diff
@@ -420,6 +413,36 @@
 ]
 
 
+# Modify numpydoc to show Attributes section like Methods section
+# PANDAS HACK: Replace attributes param_list by member_list
```

Comment:
This looks much better. But I don't think the comment is clear enough to understand what/why we are doing this. I'd probably move it to the docstring of `alternative_str` and expand it (and I'd probably rename `alternative_str` to `sphinxdocstring_str`).